packages/blade/src/components/Button/BaseButton/buttonTokens.ts

```diff
@@ -195,8 +195,8 @@ const buttonIconOnlyPadding: Record<
   xsmall: {
     top: 0,
     bottom: 0,
-    left: 4, // should be `6px` as per design but we're making it `8px` since `6px` is not available as a spacing token
-    right: 4, // should be `6px` as per design but we're making it `8px` since `6px` is not available as a spacing token
+    left: 3, // should be `6px` as per design but we're making it `8px` since `6px` is not available as a spacing token
+    right: 3, // should be `6px` as per design but we're making it `8px` since `6px` is not available as a spacing token
```

Comment:
Was it a bug before?

Reply:
yes, The comment says it should be 8px but spacing.4 refers to 12px. It should be 8px only

Reply:
Wait this are in pixels or in our spacing token scale? 

does 3 refers to spacing.3 or 3px?

Reply:
It refers to spacing.3

Reply:
hmm okay, 

nit: maybe we should instead maintain `spacing.3` instead of 3 as a number.

Reply:
yes. I have to fix that size thing for all icon buttons so I'll be creating separate PR for that